README.md

```diff
@@ -30,7 +23,139 @@ me know through the issues or discussions.
 
 ### Package installation 
 
-`go get -u github.com/Funmi4194/goinsta@latest`
+`go get -u github.com/blacheinc/goinsta`
+
+
+### Delete a comment by its specified comment ID
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/blacheinc/goinsta"
+)
+
+func main() {  
+    // import cookie 
+    insta, err := goinsta.ImportPathString("cookie")
+    if err != nil {
+		log.Fatal(err)
+	}
+
+    //retrieve postid from url
+	postId, err := GetPostId(url)
+	if err != nil {
+		log.Fatal(err)
+	}
+	mediaId, err := goinsta.MediaIDFromShortID(postId)
+	if err != nil {
+		log.Fatal(err)
+	}
+	//fetch all media
+	media, err := insta.GetMedia(mediaId)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var commentID string
```

Comment:
@Funmi4194 Where does the `commentID` come from? Here, you only initialized it but did not assign it a value at any point.